cmake/Buildergocub-software.cmake

```
@@ -12,4 +11,4 @@ ycm_ep_helper(ergocub-software
               TAG master
               COMPONENT core
               FOLDER src
-              CMAKE_ARGS ${ergocub-software_OPTIONAL_CMAKE_ARGS})
+              CMAKE_ARGS -DCOMPILE_ergoCubEmotions=BOOL:ON)
```

```suggestion
              CMAKE_ARGS -DCOMPILE_ergoCubEmotions=BOOL:ON)

set(ergocub-sofware_CONDA_DEPENDENCIES libopencv)              
```

See https://github.com/robotology/robotology-superbuild/blob/master/doc/developers-faqs.md#how-to-ensure-that-binary-packages-are-correctly-generated-for-a-new-package .